src/Processors/Transforms/Streaming/AggregatingTransform.h

```diff
@@ -257,9 +260,7 @@ class AggregatingTransform : public IProcessor
     bool read_current_chunk = false;
 
     /// Aggregated result which is pushed to downstream output
-    Chunk current_chunk_retracted;
-    Chunk current_chunk_aggregated;
-    bool has_input = false;
+    ChunkList aggregated_chunks;
```

Comment:
Since now aggregated_chunks is a list, please make sure it won't blow our memory by add more to the list, we shall always drain the aggregated chunks and then pull more from upstream